src/main/java/com/salesforce/functions/jvm/sdk/data/DataApi.java

```diff
@@ -82,30 +81,6 @@ public interface DataApi {
   @SuppressWarnings("unused")
   RecordModificationResult delete(String type, String id) throws DataApiException;
 
-  /**
-   * Creates a new and empty RecordBuilder that can be used to build a {@link Record} object for use
-   * with the {@link #update(Record)} and {@link #create(Record)} methods.
-   *
-   * @param type The type of the record to build.
-   * @return A new and empty RecordBuilder.
-   */
-  @Nonnull
-  @SuppressWarnings("unused")
-  RecordBuilder newRecordBuilder(String type);
```

Comment:
it seems like there should be only 1 way to construct a `RecordBuilder` but i'm not sure where to add that now. 
should we keep this and also provide a `BulkApi.newRecordBuilder`?